fees/usual/index.ts

```diff
@@ -0,0 +1,81 @@
+import { FetchOptions, SimpleAdapter } from "../../adapters/types";
+import { CHAIN } from "../../helpers/chains";
+import { queryDune } from "../../helpers/dune";
+const YIELD_TREASURY = "0x81ad394C0Fa87e99Ca46E1aca093BEe020f203f4";
+const USD0 = "0x73A15FeD60Bf67631dC6cd7Bc5B6e8da8190aCF5";
+const USUAL = "0xC4441c2BE5d8fA8126822B9929CA0b81Ea0DE38E";
+const USUALX = "0x06B964d96f5dCF7Eae9d7C559B09EDCe244d4B8E";
+
+const getBalanceOf = async (
+  options: FetchOptions,
+  token: string,
+  account: string
+) => {
+  const balance = await options.api.call({
+    target: token,
+    abi: "function balanceOf(address account) external view returns (uint256)",
+    params: [account],
+  });
+  return BigInt(balance);
+};
+
+const fetchTreasuryRevenue = async (options: FetchOptions) => {
+  const totalRevenue = options.createBalances();
+  const usualXHolding = await getBalanceOf(options, USUALX, YIELD_TREASURY);
+  totalRevenue.add(USUALX, usualXHolding);
```

Comment:
hmm, this is not a good way to compute total revenue since, team might deposit/withdraw tokens from this safe